packages/block-library/src/social-link/index.php

```diff
@@ -8,9 +8,9 @@
 /**
  * Renders the `core/social-link` block on server.
  *
- * @param Array   $attributes The block attributes.
- * @param String  $content InnerBlocks content of the Block.
- * @param WPBlock $block Block object.
+ * @param Array  $attributes The block attributes.
+ * @param String $content InnerBlocks content of the Block.
+ * @param Object $block Block object.
```

Comment:
```suggestion
 * @param WP_Block $block Block object.
```

I believe this is a spill over from JS types, for PHP the type should be `WP_Block` and not a generic `Object` type.
You can see `WP_Block` in use for multiple other dynamic blocks [for example here](https://github.com/WordPress/gutenberg/blob/6156f72fd556a982c16423a7c8795ba997257518/packages/block-library/src/post-comments/index.php#L13).